src/addons/Portal/Portal.js

```diff
@@ -201,6 +204,15 @@ class Portal extends Component {
   // Component Event Handlers
   // ----------------------------------------
 
+  handlePortalClick = (e) => {
+    const { closeOnPortalClick } = this.props
+
+    if (!closeOnPortalClick) return
+
+    debug('handlePortalClick()')
```

Comment:
Most of our methods have `debug()` call first. I think that this can refactored to this:

```js

handlePortalClick = (e) => {
  debug('handlePortalClick()')
  const { closeOnPortalClick } = this.props

  if (closeOnPortalClick) this.close(e)
}
```